source/features/delete-review-comments-faster.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import {observe} from 'selector-observer';
+import {TrashIcon} from '@primer/octicons-react';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import onFragmentLoad from '../github-events/on-fragment-load';
+
+const deinit: VoidFunction[] = [];
+
+async function onButtonClick({delegateTarget: button}: delegate.Event): Promise<void> {
+	select('.dropdown-menu .js-comment-delete > button', button.closest<HTMLElement>('.js-comment')!)!.click();
```

Comment:
It's best not to combine `select` with other DOM traversal because the order makes it hard to follow.

Starts from button, then goes right to `.js-comment` and then `select`’s selector is on the left.

Instead:
```suggestion
	button
		  .closest('.js-comment')!
		  .querySelector('.dropdown-menu .js-comment-delete > button')!
		  .click();
```

Also `<HTMLElement>` isn't necessary here and in the other `.closest` call below